test/submission.e2e.js

```diff
@@ -51,6 +52,9 @@ test('Happy path', async t => {
 
   // file uploads
   await t
+    // TODO re-enable this assertion once https://github.com/elifesciences/elife-xpub/issues/309 is resolved
+    // .expect(submission.verificationRibbon.exists)
```

Comment:
I suggest that we remove this line. I highly doubt that we will ever have the notification ribbon component on the dashboard. Perhaps in the manuscript preview page during QC, or perhaps not at all. It might be better to show that a Corresponding Author has/hasn't verified in some other way than the ribbon - see [this comment](https://github.com/elifesciences/elife-xpub/issues/310#issuecomment-403035189)

Reply:
Oh woops, didn't realise this was already merged